Source/WebKit/GPUProcess/graphics/RemoteGraphicsContextGL.messages.in

```diff
@@ -25,6 +25,7 @@
 #if ENABLE(GPU_PROCESS) && ENABLE(WEBGL)
 
 messages -> RemoteGraphicsContextGL NotRefCounted Stream {
+    void DrawingBufferTextureTargetAndBinding() -> (GCGLenum target, GCGLenum binding) Synchronous
```

Comment:
If you expand upwards, you can see a warning that this file should not be modified by hand.
The template is in generate-gpup-webgl script.